aten/src/ATen/cpu/vec/vec512/vec512_float.h

```diff
@@ -248,13 +248,13 @@ template <> class Vectorized<float> {
   }
   Vectorized<float> frac() const;
   Vectorized<float> sin() const {
-    return Vectorized<float>(Sleef_sinf16_u10(values));
+    return Vectorized<float>(Sleef_sinf16_u35(values));
   }
   Vectorized<float> sinh() const {
     return Vectorized<float>(Sleef_sinhf16_u10(values));
```

Comment:
Have you ever observed the same issue for `sinh`/`cosh`?

Reply:
`sinh` / `cosh` should have the same issue. However, these two ops are not included in micro-benchmarks.